Geometry/CSCGeometryBuilder/test/python/validateCSCGeometry_cfg.py

```diff
@@ -0,0 +1,56 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
This file should be called `validateCSCGeometryDD4hep_cfg.py`.

Reply:
Done